service/history/mutableStateBuilder.go

```diff
@@ -2663,6 +2664,20 @@ func (e *mutableStateBuilder) AddSignalExternalWorkflowExecutionInitiatedEvent(
 	return event, si, nil
 }
 
+func (e *mutableStateBuilder) AddUpsertWorkflowSearchAttributesEvent(
```

Comment:
plz format like this
```
func (e *mutableStateBuilder) AddSignalExternalWorkflowExecutionInitiatedEvent(
	decisionCompletedEventID int64,
	signalRequestID string,
	request *workflow.SignalExternalWorkflowExecutionDecisionAttributes,
) (*workflow.HistoryEvent, *persistence.SignalInfo, error) {
```

Reply:
and could you put AddUpsertWorkflowSearchAttributesEvent and ReplicateUpsertWorkflowSearchAttributesEvent close together?

Reply:
updated, make sense